src/test/java/organice/model/person/DonorTest.java

```diff
@@ -133,7 +133,7 @@ public void toStringTest() {
         assertEquals(john.toString().trim(),
             "John Person Type: donor Nric: T1312123P Phone: 81230942 Age: 60 "
                         + "Blood Type: A Tissue Type: 1,2,3,4,5,6 Organ: kidney Organ "
-                                + "Expiry Date: 20-Jan-2020");
+                                + "Expiry Date: 20-Jan-2020 " + "Status: not processing");
```

Comment:
Minor issue, but it could be changed to this: "Expiry Date: 20-Jan-2020 Status: not processing"

Reply:
amended